lib/prepare-system.sh

```diff
@@ -5,6 +5,15 @@ source lib/functions.sh
 
 log "INFO" "Preparing your system for bridgehead installation ..."
 
+# Check, if running in WSL
+if [[ $(grep -i Microsoft /proc/version) ]]; then
+    # Check, if systemd is available
+    if [ $(systemctl is-system-running) -eq "offline" ]; then
```

Comment:
```suggestion
    if [ "$(systemctl is-system-running)" == "offline" ]; then
```

Reply:
IIRC, `==` only works correctly with `[[ foo ]]`, no?

Reply:
Bash never ceases to surprise but this should work with single brackets, too.